engines/ic3ia.cpp

```diff
@@ -327,13 +327,6 @@ RefineResult IC3IA::refine()
               default_random_engine(options_.random_seed_));
     }
 
-    TermVec red_preds;
-    if (ia_.reduce_predicates(cex, preds_vec, red_preds)) {
-      // reduction successful
-      preds.clear();
-      preds.insert(red_preds.begin(), red_preds.end());
-    }
-
     // add all the new predicates
     bool found_new_preds = false;
     for (auto p : preds) {
```

Comment:
this is going to add the predicates before doing the reduction. We don't want that.

Reply:
fixed in 31b219f0ea644c93f268b02052895575b6fe765c

Reply:
Oops, thanks for catching that.